uni/lib/view/home/widgets/uni_button.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:uni/utils/navigation_items.dart';
+
+class UniButton extends StatelessWidget {
+  const UniButton({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ButtonTheme(
+      materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+      shape: const RoundedRectangleBorder(),
+      child: Builder(
+        builder: (context) => TextButton(
+          onPressed: () {
+            final currentRouteName = ModalRoute.of(context)!.settings.name;
+            if (currentRouteName !=
+                '/${NavigationItem.navPersonalArea.route}') {
+              Navigator.pushNamed(
+                context,
+                '/${NavigationItem.navPersonalArea.route}',
+              );
+            } else {
+              Scaffold.of(context).openDrawer();
+            }
```

Comment:
Why is this needed? From what I can tell, the uni button only appears on the homepage and this logic just takes the user to the homepage.

Reply:
This is old code that was useful since the logo appeared on other pages and there was a drawer. I believe it can be removed now

Reply:
Should I just remove just the `onPressed` function or turn the button into an image entirelly?

Reply:
Just turn the button into an image. I think a button that does nothing is not really intuitive for the end user.